.github/workflows/unitcoverage.yaml

```diff
@@ -0,0 +1,24 @@
+name: Unit Test Coverage
+on: [push, pull_request, workflow_dispatch]
```

Comment:
Do we need `workflow_dispatch` in there?

Reply:
We do not need it, it supposed to allow someone to trigger the workflow manually and generate a code-cov report without having to submit a commit. But that functionality seems flaky on the GitHub side.

Reply:
Ohh, that's pretty nice. Cool. Thanks!